packages/app/src/app/overmind/effects/live/index.ts

```diff
@@ -80,21 +85,32 @@ export default {
         .receive('error', resp => reject(resp));
     });
   },
-  joinChannel<T>(roomId: string): Promise<T> {
-    const { socket } = this.context;
-
-    channel = socket.getSocket().channel(`live:${roomId}`, {});
+  joinChannel(
+    roomId: string
+  ): Promise<{
```

Comment:
I think it's best to extract this in a separate type so you don't need the `as any`, but can do `as type`.

Reply:
Yeah, I fixed it, but for Typescript it does not actually matter. If you set an explicit return type on a method, that return type will be used even if you resolve/return an `any` :)